### DISCUSSION TITLE
可否分享一下README里面示例图的生成参数？

### DISCUSSION THREAD

**Discussion by aliencaocao** [koishijs/novelai-bot#87] [Q&A] [answered]
很喜欢里面的几张，谢谢

**Comment by shigma** [comment, marked as answer]
很高兴你能喜欢，不过前面几张太久远我都找不到 prompt 了。:rofl:

PS：如果有你觉得做得好的图也欢迎告诉我，我可以加进 readme！当然，得是用这个 bot 做的。

**Reply by shigma** [reply]
![](https://cdn-shiki.momobako.com:444/static/portrait/a31uu-ou34k.webp)

1girl, loli, blue eyes, blonde long hair, flat chest, white pantyhose, blush, {{see through}}, legs together, light blue sleeveless dress, wet, sit in the water, pond, extremely detailed CG unity 8k wallpaper, no shoes, detached sleeves, collarbone, detached sleeves, blue ribbon

**Reply by shigma** [reply]
![](https://cdn-shiki.momobako.com:444/static/portrait/a7k8p-gba0y.webp)

one girl, pink hair, long hair, blue eyes, blush, shy, blue sailor collar, black short school uniform, navel, open mouth, blue miniskirt, hands together, heart hands, masterpiece, best quality
